draft-ietf-tls-mldsa.md

```diff
@@ -86,16 +86,22 @@ These correspond to ML-DSA-44, ML-DSA-65, and ML-DSA-87 defined
 in {{FIPS204}} respectively. Note that these are different
 from the HashML-DSA pre-hashed variantsadefined in Section 5.4 of {{FIPS204}}.
 
-The signature MUST be computed and verified as specified in
+If one of those SignatureSchemes values is used in a CertificateVerify message,
+then the signature MUST be computed and verified as specified in
 {{Section 4.4.3 of RFC8446}}.
 
 The context parameter defined in {{FIPS204}} Algorithm 2 and 3
 MUST be the empty string.
 
-The corresponding end-entity certificate when negotiated MUST
+When used in CertificateVerify, the corresponding end-entity certificate MUST
 use id-ML-DSA-44, id-ML-DSA-65, id-ML-DSA-87 respectively as
 defined in {{I-D.ietf-lamps-dilithium-certificates}}.
```

Comment:
Perhaps better to merge the paragraphs as

> If one of those SignatureSchemes values is used in a CertificateVerify message,
> then the signature MUST be computed and verified as specified in
> {{Section 4.4.3 of RFC8446}}, and the corresponding end-entity certificate MUST
> use id-ML-DSA-44, id-ML-DSA-65, id-ML-DSA-87 respectively as
> defined in {{I-D.ietf-lamps-dilithium-certificates}}.

Reply:
yes, that does sound better